libs/build-nss-desktop.sh

```diff
@@ -59,7 +59,7 @@ fi
 if [[ "${CROSS_COMPILE_TARGET}" =~ "darwin" ]]; then
   #From https://firefox-ci-tc.services.mozilla.com/tasks/index/app-services.cache.level-3.content.v1.nss-artifact/latest
   curl -sfSL --retry 5 --retry-delay 10 -O "https://firefox-ci-tc.services.mozilla.com/api/index/v1/task/app-services.cache.level-3.content.v1.nss-artifact.latest/artifacts/public/dist.tar.bz2"
-  SHA256="9573e640797200891dadc6401cdd9cb06e3fd48518a146cf2f578e83127cb7f2"
+  SHA256="15e8cab63a4e0484564231a24d23d8be349217ba6add53b56609c339bb306829"
   echo "${SHA256}  dist.tar.bz2" | shasum -a 256 -c - || exit 2
```

Comment:
I guess the task is failing because the index hasn't been updated yet?

Reply:
I think the long-term solution to this is get in the monorepo.  In the meantime, what if we added support for overriding the URL via an ENV variable?  Then when we're running level 1 builds, we use the level 1 index.